src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,67 @@
+package command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import exception.InvalidDateTimeFormatException;
+import exception.InvalidInputException;
+import exception.InvalidSaveFileException;
+import logic.Storage;
+import logic.Ui;
+import tasks.Deadlines;
+import tasks.TaskList;
+
+/**
+ * Adds the deadline entry that the user input to the
+ * Arraylist of logic.Duke.
+ */
+public class DeadlineCommand extends Command {
+
+    private static DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    public DeadlineCommand(String input) {
+        super(input);
+    }
+
+    /**
+     * Add deadline entry to the Arraylist.
+     *
+     * @param tasks List of tasks given.
+     * @param ui Handles the output to print.
+     * @param storage Writes the save file.
+     * @throws InvalidDateTimeFormatException If input does not follow format specified.
+     * @throws InvalidInputException If the input for the delete is incorrect.
+     * @throws InvalidSaveFileException If there is an issue writing the save file.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws InvalidDateTimeFormatException,
+            InvalidInputException, InvalidSaveFileException {
+        if (super.input.length() <= 9) {
```

Comment:
Would `9` be better abstracted out as a constant? I see something similar in the other command classes as well